config/helpers.php

```diff
@@ -3,6 +3,7 @@
 use Kirby\Cms\App;
 use Kirby\Cms\Helpers;
 use Kirby\Cms\Html;
+use Kirby\Cms\Pages;
```

Comment:
Can be removed now I think:

```suggestion
```

Reply:
Off, dammit - always the little things :D